### DISCUSSION TITLE
Using in Svelte?

### DISCUSSION THREAD

**Discussion by ysk3a** [skyra-project/discord-components#370] [Q&A] [answered]
Cool components!

I think this is more appropriate to ask in discusion but any chance you could add how to use in svelte/sveltekit?
Or maybe if someone in the community successfully was able to use the web components in svelte could share how about I could add to a svelte project? It looks like no styles are added.

**Comment by favna** [comment, marked as answer]
It is actually very similar to Vue because Svelte's syntax in general is similar to Vue. Here is a codesandbox that uses Codesandbox' template for Svelte: https://codesandbox.io/s/discord-components-svelte-dw6fhw?file=/App.svelte

Edit: how it works for sveltekit though I don't know because I cannot find any documentation for sveltekit for how to define custom webcomponents. Note that this will therefore not be an issue on this library, but rather one on sveltekit. I'm sure there must be a way to use webcomponents in sveltekit but it's not clearly documented.
